components/director/pkg/graphql/schema.graphql

```diff
@@ -742,6 +742,7 @@ type Auth {
 	additionalQueryParams: QueryParams
 	additionalQueryParamsSerialized: QueryParamsSerialized
 	requestAuth: CredentialRequestAuth
+	oneTimeToken: OneTimeTokenForApplication
```

Comment:
In my opinion, this should work for all kind of OTT, not only Application, as the type is Auth, which is generic.